### DISCUSSION TITLE
WIFI RTL8188GU 802.11n WLAN Adapter Support

### DISCUSSION THREAD

**Discussion by linhungsam** [OpenNeptune3D/OpenNept4une#192] [Q&A]
Hi,

Does OpenNept4une support this wifi adapter RTL8188GU? lsusb command shows the device but nmtui doesn't detect any wifi signal, manually adding wifi acording to the wifi setup guide doesn't work. "Ip a" command only show ethernet connection. The stock firmware supports this adapter out of box, and nmtui can detect wifi signal without manually adding my home wifi.

Any idea? do I need to install extra drivers?

Thanks,

Sam

**Comment by halfmanbear** [comment]
https://github.com/OpenNeptune3D/OpenNept4une/blob/main/supported-wifi-adapters

We don't support that specific adapter because the bodged Linux driver for it uses 25% of the CPU and causes issues. MKS deliberately excluded it from their boards although Elegoo re-introduced it (not a good idea)
